mama/c_cpp/src/c/dqpublishermanager.c

```diff
@@ -557,3 +557,20 @@ mama_status mamaDQPublisherManager_sendNoSubscribers (
 
     return MAMA_STATUS_OK;
 }
+
+mama_status mamaDQPublisherManager_getPublisher (
+        mamaDQPublisherManager manager,
+        const char *symbol,
+        mamaDQPublisher* pub)
+{
+    mamaDQPublisherManagerImpl* impl  = (mamaDQPublisherManagerImpl*) manager;
+    mamaPublishTopic* topic           = NULL;
```

Comment:
Is there some reason to initialize the topic variable to NULL before assigning it to the result of wtable_lookup?
Why not initialize it to to the result of wtable_lookup if there is no such reason?